setup.py

```diff
@@ -54,7 +55,7 @@
       use_katversion=True,
       install_requires=['redis>=3.3', 'six>=1.12', 'netifaces',
                         'msgpack', 'numpy'],
-      extras_require={'rdb': ['rdbtools', 'python-lzf']},
-      tests_require=['rdbtools', 'fakeredis[lua]>=1.3.1'],
+      extras_require={'rdb': ['rdbtools', 'python-lzf'], 'test': tests_require},
```

Comment:
Is there value in making async an extra as well ? We could then relax the Python 3.6 requirement for a basic install.

Reply:
I did that in a later commit.